dll/win32/comctl32/comctl32.h

```diff
@@ -88,6 +88,9 @@ extern HBRUSH  COMCTL32_hPattern55AABrush DECLSPEC_HIDDEN;
 
 #define IDT_CHECK        401
 
+/* Command Link arrow */
+#define IDB_CMDLINK      402
```

Comment:
Is it just me or is this image not actually provided anywhere?

Reply:
I agree that it does seem to be missing from the wine sync after doing a search.
But it is available here: https://github.com/wine-mirror/wine/blob/master/dlls/comctl32/idb_cmdlink.bmp
Hopefully, it will eventually get added into this sync.

Reply:
Moreover, `CL_Paint()`, which uses this define/file, is missing too.
https://github.com/search?q=repo%3Awine-mirror%2Fwine%20IDB_CMDLINK&type=code

Read `exception of button.c which all in all is a massive fork over wines code`.
Then, simply `#ifndef` (with a comment) these 2 declarations?

Reply:
We can just import this function It's fine, Ill look into it today

Reply:
> import this function

Wouldn't that be bloat, unless this function is actually used?

Reply:
Well the icon is here now, but im not going to try and rewrite button.c here. Ill leave it as is so we can move on